frontend/packages/dev-console/src/components/NamespacedPage.tsx

```diff
@@ -34,7 +34,7 @@ const NamespacedPage: React.FC<NamespacedPageProps> = ({
       onNamespaceChange={onNamespaceChange}
       hideProjects={hideProjects}
     >
-      {!hideApplications && <ApplicationSelector disabled={disabled} />}
+      {!hideApplications && <NamespaceBarApplicationSelector disabled={disabled} />}
```

Comment:
There were two different ApplicationSelector components, one in shared one in dev-console. Both are now in topology (since the use the topology data retrieval) and this one was renamed.

Reply:
Gotcha. Thanks for the clarification!